README.md

```diff
@@ -18,9 +18,19 @@ curl --location --request POST '<base_url>:8080/token' \
     "projectid": "<projectid>"
 }'
 ```
+where `<basic_auth_from_creds>` is the base64 encoded string `username:password`.
 
 The `token` endpoint requires basic auth and the allowed credentials can be defined in the configuration file `config.yaml`.
 
+ex.
+```bash
+$ curl --location --request POST 'localhost:8080/token' \
+       --header "Authorization: Basic $(printf 'uppmax:uppmax' | base64)" \
+       --header 'Content-Type: text/plain' \
```

Comment:
Maybe it doesn't make a difference. but that's how I had it in postman, so adding it here
```suggestion
       --header 'Content-Type: application/json' \
```